package.yaml

```diff
@@ -69,7 +72,7 @@ library:
         - dump-core
       ghc-options:
         - -fplugin=DumpCore
-        - -fplugin-opt DumpCore:core-html
+        - -fplugin-opt=DumpCore:core-html
```

Comment:
I am reasonably sure this works without the change

Reply:
I wanted to be consistent and `=` was just my personal preference - should I discard this change?

Reply:
I'm happy with it so long as it works.